recipes/bspline/LICENSE.md

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```

Comment:
This file is already included in the source distribution, so you don't need to include it again here.

Reply:
HI, thanks for reviewing this. I just removed the LICENSE.md file, and the license_file entry in meta.yaml, but now the linter complains that that entry is needed. If I put the entry back without the file itself, the builder throws an error. What's the best way to do this?